content/reference/sql/supported-clients/sql-editors.md

```diff
@@ -7,40 +7,101 @@ title: SQL Editors
 Dolt comes with a built in MySQL compatible server that you can connect SQL Editors too. Here are a list of notable MySQL Editors and our compatibility status.
 | Editor | Supported | Notes and limitations |
 | :--- | :--- | :--- |
-| [Tableplus](https://tableplus.com/) | ✅ | |
+| [Tableplus](https://tableplus.com/) | ✅ | Recommended (see below) |
 | [Navicat For Mysql](https://www.navicat.com/en/products/navicat-for-mysql) | ✅ | |
 | [Dbeaver](https://dbeaver.io/) | ✅ | |
-| [Datagrip](https://www.jetbrains.com/datagrip/) | ✅ | Need to implement a couple functions |
+| [Datagrip](https://www.jetbrains.com/datagrip/) | ✅ | |
 | [MySQL Workbench](https://www.mysql.com/products/workbench/) | ❌ | Missing some information schema materials |
 
-## Recommended Dolt Configuration
+## Setting up Dolt with Tableplus
 
-Some editors require custom configuration of dolt's sql-server. For example, DBeaver use multiple connections to run queries whereas `dolt sql-server` supports 1 connection out of the box. Here's a sample config file that should get you started with `dolt sql-server`
+[Tableplus](https://tableplus.com/) is the recommended SQL editor to use with dolt. Let's clone a Dolt repo and connect it to Tableplus.
 
-<div class="gatsby-highlight" data-language="text">
-	<pre class="By default, starts a MySQL-compatible server whilanguage-text">
-		<code class="language-text">
+### Step 1: Cloning a repository.
 
-log_level: trace
+Open your terminal and clone the following [repository](https://www.dolthub.com/repositories/dolthub/ip-to-country/data/master).
 
-behavior:
-read_only: false
-autocommit: true
+```bash
+dolt clone dolthub/ip-to-country && cd ip-to-country
+```
+
+Now run: `dolt sql -q 'show tables'` and you should see the following output
+
+```bash
+> dolt sql -q 'show tables'
++---------------+
+| Table         |
++---------------+
+| IPv4ToCountry |
+| IPv6ToCountry |
++---------------+
+```
+
+### Step 2: Setting up your server
+
+Some editors require custom configuration of dolt's [sql-server](https://docs.dolthub.com/reference/cli#dolt-sql-server). For example, DBeaver use multiple connections to run queries whereas `dolt sql-server` supports 1 connection out of the box. Here's a sample config file that should get you started with `dolt sql-server`. Save this in the `ip-to-country` directory we created above as `config.yaml`
```

Comment:
It's not true that dolt only supports 1 connection anymore (hasn't been true since we enabled transactions)

https://docs.dolthub.com/reference/cli#dolt-sql-server

Reply:
Didn't realize the default for max connections got updated.